x-pack/test/security_solution_api_integration/config/services/security_solution_serverless_utils.ts

```diff
@@ -49,6 +49,8 @@ export function SecuritySolutionServerlessUtils({
 
   return {
     getUsername: async (role = 'admin') => {
+      // load service to call it outside mocha context
+      await svlUserManager.init();
```

Comment:
Since I changed service function to be `async`, we need to call `init()` to actually load it.